src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.cs

```diff
@@ -25,13 +25,15 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
     using Workspace = Microsoft.CodeAnalysis.Workspace;
 
     [Export(typeof(MiscellaneousFilesWorkspace))]
-    internal sealed partial class MiscellaneousFilesWorkspace : Workspace, IVsRunningDocTableEvents2
+    internal sealed partial class MiscellaneousFilesWorkspace : Workspace
     {
         private readonly IVsEditorAdaptersFactoryService _editorAdaptersFactoryService;
         private readonly IMetadataAsSourceFileService _fileTrackingMetadataAsSourceService;
         private readonly IVsRunningDocumentTable4 _runningDocumentTable;
         private readonly IVsTextManager _textManager;
 
+        private readonly RunningDocumentTableEventSink _runningDocumentTableEventSink;
```

Comment:
I don't think it is Sink anymore. something like RDTTracker or something?

Reply:
Sure, I guess since the events are getting forwarded sink doesn't make a lot of sense.  Will update!